### DISCUSSION TITLE
2023/05/25/how-to-install-thanos/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#164] [Blog Comments]
# 2023/05/25/how-to-install-thanos/

급하신 분들은 설치로 바로 가시면 됩니다. Prometheus의 저장소 및 HA 문제 Prometheus도 remote storage에 저장하는 기능을 제공하긴 하지만

https://tech.cloudmt.co.kr/2023/05/25/how-to-install-thanos/

**Comment by wooyoungAhn** [comment]
좋은 글 감사합니다.
혹시 thanos-query-frontend 구성에 cashe도 따로 설정하셨나요?
최근에 모니터링 대상이 많아지면서 쿼리 속도 개선 방안을 찾고 있습니다. 
어떻게 구성했는지 설정 공유 가능할까요?
- [thanos query-frontend ](https://thanos.io/tip/components/query-frontend.md/)
